doctr/datasets/loaders/detection/core.py

```diff
@@ -84,7 +84,7 @@ def on_epoch_end(self):
     def __getitem__(
         self,
         index: int
-    ) -> Tuple[tf.Tensor, tf.Tensor, tf.Tensor]:
+    ) -> Tuple[Any]:
```

Comment:
Mmmh, perhaps when we use the inheritance, this will have to change, but in this PR, I think that 
`) -> Tuple[tf.Tensor, tf.Tensor, tf.Tensor]:` is the only correct possibility